### PR TITLE
Document how to setup event subscriptions via kyma CLI

### PR DESCRIPTION
**Pull request by pPrecel**
**Description**

Changes proposed in this pull request:

- Add the new section in getting-started guides about subscribing a function using the Kyma CLI (  `Trigger the Function with an event` )

**Related issue(s)**

https://github.com/kyma-project/cli/issues/824

### CODE REVIEW

File: docs/02-get-started/13-trigger-function-with-event.md [hunk 1]
```diff
@@ -71,6 +71,53 @@ Console UI
...
+                    property: source
+                    type: exact
+                    value: ""
+                  eventType:
+                    property: type
+                    type: exact
+                    value: sap.kyma.custom.commerce.order.deliverysent.v1
+    ```
+
+3. Run the apply Kyma CLI command to create a Function CR and a Subscription CR in the YAML format on your cluster:
```
Comment (mmitoraj) [inline docs/02-get-started/13-trigger-function-with-event.md:109]:
```suggestion
3. Run the 'apply' Kyma CLI command to create a Function CR and a Subscription CR in the YAML format on your cluster:
```

File: docs/02-get-started/13-trigger-function-with-event.md [hunk 2]
```diff
@@ -71,6 +71,53 @@ Console UI
...
+
+    ```bash
+    kyma apply function
+    ```
+
+4. Check that the Subscription CR was created and is ready. This is indicated by its status equal to `true`:
```
Comment (mmitoraj) [inline docs/02-get-started/13-trigger-function-with-event.md:115]: Should the status be 'true' or 'ready'. Sorry if it's a silly question but this part was misleading for me `{.status.ready}`.

**Review by mmitoraj — COMMENTED** [review]
Just a few comments from me :)

File: docs/02-get-started/13-trigger-function-with-event.md (hunk 2 above)
Comment (pPrecel) [reply docs/02-get-started/13-trigger-function-with-event.md:115]: Everything is ok. It means that output from the spell described below should be exactly `True`. And this `True` means that everything is ready to use.

In the CR (yaml) it looks like this:

```yaml
...
status:
    ready: true
...
```

File: docs/02-get-started/13-trigger-function-with-event.md [hunk 3]
```diff
@@ -71,6 +71,53 @@ Console UI
 
 A message confirming that the Subscription was created will appear in the **Event Subscriptions** section in the Function's details view.
 
+  </details>
+<details>
+<summary label="kyma-cli">
+Kyma CLI
+</summary>
+
+1. Run the 'init' Kyma CLI command to create local files with the default configuration. Go to the folder in which you want to initiate the workspace content and run this command:
```
Comment (mmitoraj) [inline docs/02-get-started/13-trigger-function-with-event.md:80]:
```suggestion
1. Run the `init` Kyma CLI command to create local files with the default configuration. Go to the folder in which you want to initiate the workspace content and run this command:
```

File: docs/02-get-started/13-trigger-function-with-event.md [hunk 4]
```diff
@@ -71,6 +71,53 @@ Console UI
...
+                    property: source
+                    type: exact
+                    value: ""
+                  eventType:
+                    property: type
+                    type: exact
+                    value: sap.kyma.custom.commerce.order.deliverysent.v1
+    ```
+
+3. Run the 'apply' Kyma CLI command to create a Function CR and a Subscription CR in the YAML format on your cluster:
```
Comment (mmitoraj) [inline docs/02-get-started/13-trigger-function-with-event.md:109]:
```suggestion
3. Run the `apply` Kyma CLI command to create a Function CR and a Subscription CR in the YAML format on your cluster:
```

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**